### ISSUE TITLE
第44回　コンピュータビジョン勉強会＠関東

### ISSUE THREAD

**Issue by kmotohas** [kmotohas/dl4me#4]
https://kantocv.connpass.com/event/76984/

**Comment by kmotohas** [comment]
![image](https://user-images.githubusercontent.com/7589812/35953587-a58cf29e-0cc8-11e8-8a0f-df5ff4bd692c.png)

**Comment by kmotohas** [comment]
Curiosity driven exploration
https://www.slideshare.net/takmin/curiosity-driven-exploration
Visual Forecasting by Imitating Dynamics in Natural Sequences (ICCV’17) 論文紹介
https://speakerdeck.com/jellied_unagi/visual-forecasting-by-imitating-dynamics-in-natural-sequences-iccv-17-lun-wen-shao-jie
ビジョンとロボットの強化学習
https://www.slideshare.net/yasunoriozaki12/ss-87188017
Cold-Start Reinforcement Learning with Softmax Policy Gradient
https://www.slideshare.net/FujimotoKeisuke/coldstart-reinforcement-learning-with-softmax-policy-gradient
20180204 reinforcement learning
https://www.slideshare.net/secret/viWVOisRb70bOf
180204 Attention-aware Deep Reinforcement Learning for Video Face Recognition
https://www.slideshare.net/takanoriogata1121/180203-attentionaware-deep-reinforcement-learning-for-video-face-recognition
Sequence Level Training with Recurrent Neural Networks （関東CV勉強会 強化学習論文読み会）
https://www.slideshare.net/YoshitakaUshiku/sequence-level-training-with-recurrent-neural-networks-cv/
